extensions/positron-python/resources/walkthrough/interpreterManagement.md

```diff
@@ -0,0 +1,7 @@
+Interpreters are the Python runtimes that Positron uses to run your code.
+Your interpreter can be a virtual environment such as `venv`, `conda`, or `pyenv`, or another Python installation.
+
+A few common tasks you might want to do with interpreters include:
+- **Select an interpreter**: You can select a specific interpreter for your project by using the command [Python: Select Interpreter](command:workbench.action.language.runtime.selectSession). This will show you a list of available interpreters, and you can choose the one you want to use.
```

Comment:
```suggestion
- **Select an interpreter**: You can select a specific interpreter for your project by using the command ["Select Interpreter Session"](command:workbench.action.language.runtime.selectSession). This will show you a list of available interpreters, and you can choose the one you want to use.
```

It doesn't say Python there.